src/wizard/FeedbackMessages.tsx

```diff
@@ -0,0 +1,30 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+
+export const FeedbackMessages = ({ messages }: FeedbackMessagesProps) => {
```

Comment:
[minor] Lately, we are using `React.FC<Props>` signature, it will make sure the returned value makes sense.